VideoLocker/src/test/java/org/edx/mobile/util/VersionTest.java

```diff
@@ -0,0 +1,143 @@
+package org.edx.mobile.util;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+/**
+ * Tests for verifying implementation correctness of {@link Version}.
+ */
+public class VersionTest {
+    /**
+     * Verify that a new instance created with a two-dot release
+     * string (x.x.x) is parsed correctly, setting the major,
+     * minor, and patch version properties.
+     */
+    @Test
+    public void newInstance_withTwoDotRelease_isParsedCorrectly() {
```

Comment:
Tests in this class are not following [Android's Guidelines for naming unit tests](https://source.android.com/source/code-style.html#javatests-style-rules).